README.rst

```diff
@@ -1432,18 +1341,47 @@ Example usage:
 Monitoring scripts
 ------------------
 
-The monitoring scripts which are automatically installed by a `migration file of device-monitoring app <https://github.com/openwisp/openwisp-monitoring/blob/master/openwisp_monitoring/device/migrations/0002_create_template.py>`_
-are required to make the `checks <https://github.com/openwisp/openwisp-monitoring#available-checks>`_ and
-`metrics <#openwisp_monitoring_metrics>`_ work.
+Monitoring scripts are now deprecated in favour of `monitoring packages <https://github.com/openwisp/openwrt-openwisp-monitoring#openwrt-openwisp-monitoring>`_.
+Follow the migration guide in `Migrating from monitoring scripts to monitoring packages <#migrating-from-monitoring-scripts-to-monitoring-packages>`_
+section of this documentation.
+
```

Comment:
I have kept this section in the documentation for returning users. I think this has better DX than completely removing this section.